docs/tracking.md

```diff
@@ -104,7 +104,7 @@ please open an issue on [this issue tracker](https://github.com/Quansight-Labs/f
 | PyTorch              |      [:simple-github:](https://github.com/pytorch/pytorch/issues/130249)       | :material-check-bold: |   :material-check-bold:    |           2.6.0            | :simple-linux: :simple-apple: :material-microsoft-windows: |          [:simple-pytorch:](https://pytorch.org/get-started/locally/#start-locally)          |
 | PyWavelets           |                                                                                | :material-check-bold: |   :material-check-bold:    |           1.7.0            | :simple-linux: :simple-apple: :material-microsoft-windows: |    [:simple-anaconda:](https://anaconda.org/scientific-python-nightly-wheels/pywavelets/)    |
 | pywinpty             |        [:simple-github:](https://github.com/andfoy/pywinpty/issues/491)        | :material-check-bold: |   :material-check-bold:    |           2.0.15           |                                                            |                                                                                              |
-| PyYAML               |          [:simple-github:](https://github.com/yaml/pyyaml/issues/856)          |                       |                            |                            |                                                            |                                                                                              |
+| PyYAML               |          [:simple-github:](https://github.com/yaml/pyyaml/issues/856)          |                       |                            |     6.0.3 (only 3.14+)     | :simple-linux: :simple-apple: :material-microsoft-windows: |                                                                                              |
```

Comment:
I think you can delete the (only 3.14+), we're not explicitly tracking that here and I think it will very quickly become stale information as people migrate from 3.13t.